__tests__/commands/workspaces.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import {BufferReporter} from '../../src/reporters/index.js';
+import {run as workspace} from '../../src/cli/commands/workspaces.js';
+import * as reporters from '../../src/reporters/index.js';
+import Config from '../../src/config.js';
+import path from 'path';
+
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'workspace');
+
+async function runWorkspaces(
+  flags: Object,
+  args: Array<string>,
+  name: string,
+  checkSteps?: ?(config: Config, reporter: BufferReporter) => ?Promise<void>,
+): Promise<void> {
+  const cwd = path.join(fixturesLoc, name);
+  const reporter = new reporters.BufferReporter({stdout: null, stdin: null, isSilent: true});
+
+  const config = await Config.create({cwd}, reporter);
+
+  await workspace(config, reporter, flags, args);
+
+  if (checkSteps) {
+    await checkSteps(config, reporter);
+  }
+}
+
+test('workspaces should list the workspaces', (): Promise<void> => {
```

Comment:
Test name is misleading now. It should say `workspaces info should list the workspaces`